client/src/components/graph/graph.js

```diff
@@ -261,7 +264,10 @@ class Graph extends React.Component {
       const { drawPoints, pointBuffer, colorBuffer, flagBuffer } = this.state;
 
       if (hasResized) {
-        projectionTF = createProjectionTF(this.reglCanvas.width, this.reglCanvas.height);
+        projectionTF = createProjectionTF(
```

Comment:
What line length are we splitting at?  Originally, I thought we used 120 characters as we all have big screens, but this looks like it is splitting on a much shorter length.

Reply:
[Prettier defaults to a length of 80](https://prettier.io/docs/en/options.html#print-width)

Reply:
ah yes, i am confusing it with our Python standards.   You might poll the team on what line length they prefer, but for now, leaving it with the default is sensible.